llama/utils.py

```diff
@@ -0,0 +1,37 @@
+import platform
+import torch
+
+# setting False since MPS not yet supported BFloat16 that is required for LLama2
+enable_mps = False
+
+
+def is_it_apple_arm():
+    if platform.system() != 'Darwin':
+        return False
+    if platform.machine() != 'arm64':
+        return False
+    return True
+
+
+def distrubuted_device():
```

Comment:
Small typo, I recommend to refactor it

```suggestion
def distributed_device():
```

[here](https://github.com/facebookresearch/llama/pull/504/files#diff-cfa9e089ebf97ee064b754a2fecad8b5c23610a519a6d640710d94f74fbd8300R10)
[and here](https://github.com/facebookresearch/llama/pull/504/files#diff-cfa9e089ebf97ee064b754a2fecad8b5c23610a519a6d640710d94f74fbd8300R10)